nixos/modules/services/misc/matrix-synapse.nix

```diff
@@ -552,16 +551,6 @@ in {
           accessible to anonymous users.
         '';
       };
-      trusted_third_party_id_servers = mkOption {
```

Comment:
Please use `mkRemovedOptionModule` here with some instructions what to use instead, otherwise it might be pretty confusing if one option simply went missing :)

Reply:
Is that correct ?